core/README.md

```diff
@@ -27,3 +27,17 @@ poetry run python -m extract_data.cli --help
 # Extract the Nix data for nixpkgs on x86_64-linux distributions
 poetry run python -m extract_data.cli
 ```
+
+To extract the data from a small flake, for example, `github:edolstra/dwarffs` you can run the following while in the repository Nix shell:
+
+
+```bash
+# Install dependencies
+poetry install
+
+# View help menu
+poetry run python -m extract_data.core --help
+
+# Extract the Nix data for github:edolstra/dwarffs on x86_64-linux distributions
+poetry run python -m extract_data.core --target-flake-ref="github:edolstra/dwarffs"
+```
```

Comment:
Why was this introduced? The CLI instructions are available above

Reply:
It was just for demonstrating the data extraction, writing to disk and validation process, will be potentially removed afterwards :smiley: .